### PR TITLE
ResourceGroup implementation of Provider interface

### PR DESCRIPTION
**Pull request by seans3**
* `ResourceGroup` implementation of the `Provider` interface
* `ResourceGroup` version of `InventoryClient`
* `ResourceGroup` version of `ManifestReader`

### CODE REVIEW

File: internal/live/rgprovider.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+func (f *ResourceGroupProvider) ToRESTMapper() (meta.RESTMapper, error) {
+	return f.factory.ToRESTMapper()
+}
+
+// ManifestReader returns the ResourceGroup inventory object version of
+// the ManifestReader.
+func (f *ResourceGroupProvider) ManifestReader(reader io.Reader, args []string) manifestreader.ManifestReader {
+	namespace, enforceNamespace, err := f.factory.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		namespace = metav1.NamespaceDefault
```
Comment (mortent) [inline internal/live/rgprovider.go:50]: Why continue here? Shouldn't we just return an error?

Comment (seans3) [reply internal/live/rgprovider.go]: Fixed.

**Review by mortent — APPROVED** [review]

**Pull request merged**